common/proto/evm.token.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+
+package messari.evm.token.v1;
+
+message Tokens {
+  repeated Token items = 1;
+}
+
+message Token {
+  string address = 1;
+  string name = 2;
+  string symbol = 3;
+  uint64 decimals = 4;
+  optional string total_supply = 5;
+}
+
+message Transfers {
+  repeated Transfer items = 1;
+}
+
+message Transfer {
+  string tx_hash = 1;
+  optional uint32 log_index = 2;
+  optional uint64 log_ordinal = 3;
+  Token token = 4;
+  string from = 5;
+  string to = 6;
+  string amount = 7; // BigInt, in token's native amount
+  optional string amount_usd = 8;
+  optional string reason = 9;
+}
+
+message TokenBalance {
```

Comment:
I'm not sure what's your conceptual representation of a `TokenBalance`. Maybe it'll be helpful to document it. It seems to me that this is a spot balance which is specific to a particular block/timestamp. If that's the case I'd include block height and timestamp in the message as well

Reply:
included